trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? name[0, 10] : name
+  end
```

Comment:
- [ ] Kindly make sure the `TrimmerDecorator.correct_name` method is able to return the name which has a maximum of 10 characters. If it's longer it should trim the word. Right now here is the error I am getting when trying to test it:
![Screenshot from 2022-08-25 03-15-57](https://user-images.githubusercontent.com/83042742/186552455-25bfed32-5605-4327-8f0b-1977b3eb2836.png)